src/app/(views)/managed-courses/page.tsx

```diff
@@ -41,62 +53,112 @@ export default function ManagedCourses() {
+            <>
               {courses.map((course) => (
-                <CTableRow key={course.id}>
-                  <CTableDataCell>
-                    <Link href={`/managed-courses/${course.id}`}>{course.id}</Link>
-                  </CTableDataCell>
-                  <CTableDataCell>{course.title}</CTableDataCell>
-                  <CTableDataCell>{course.description}</CTableDataCell>
-                  <CTableDataCell>
-                    <CButton
-                      color="primary"
-                      variant="outline"
-                      size="sm"
-                      href={`/managed-courses/${course.id}/edit`}
-                    >
-                      Edit
-                    </CButton>
-                    <CButton
-                      color="danger"
-                      variant="outline"
-                      size="sm"
-                      onClick={() => deleteCourse(course.id)}
-                      disabled={deletingCourse === course.id}
-                    >
-                      Delete
-                    </CButton>
-                  </CTableDataCell>
-                </CTableRow>
+                <>
+                  <CCol key={course.id}>
+                    <CCard className="h-100">
+                      <Link
+                        href={`/managed-courses/${course.id}`}
+                        style={{ textDecoration: 'none' }}
+                      >
+                        <CCardImage orientation="top" src="/images/react.jpg" alt={course.title} />
+                      </Link>
+                      <CCardBody className="d-flex flex-column">
+                        <Link
+                          href={`/managed-courses/${course.id}`}
+                          style={{ textDecoration: 'none', color: 'inherit' }}
+                        >
+                          <CCardTitle className="text-dark-emphasis" style={{ fontSize: '1rem' }}>
+                            {course.title}
+                          </CCardTitle>
+                          <CCardText
+                            className="text-secondary text-truncate mb-2"
+                            style={{ fontSize: '0.9rem' }}
+                          >
+                            {course.description}
+                          </CCardText>
+                        </Link>
+                        <div className="d-flex justify-content-end mt-auto">
+                          <Link
+                            href={`/managed-courses/${course.id}`}
+                            className="me-2 d-flex align-items-center gap-1"
+                            style={{ textDecoration: 'none' }}
+                          >
+                            <small className="text-secondary d-none d-sm-inline">View Course</small>
+                            <i className="bi bi-chevron-right text-secondary"></i>
+                          </Link>
+                          {/* <CDropdown alignment="end">
```

Comment:
Do we need this commented code? if yes, put a comment above explaining why this is still here.